src/collection.ts

```diff
@@ -1535,15 +1531,6 @@ export class Collection<TSchema extends Document = Document> {
     return new OrderedBulkOperation(this as TODO_NODE_3286, resolveOptions(this, options));
   }
 
-  /** Get the db scoped logger */
-  getLogger(): Logger {
```

Comment:
Looks like this API and the one below were never marked deprecated in 4.x, they are implicitly deprecated because they return a deprecated object, but it might be good to have them explicitly deprecated, too. Can we file a ticket to deprecate these in 4.x? Bailey will be backporting some tests into 4.x anyway, so we can cut and release a new minor next week.